octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -256,7 +268,52 @@ def convert_currency_value_using_last_prices(self, quantity, current_currency, t
             pass
         except KeyError:
             pass
-        raise errors.MissingPriceDataError(f"no price data to evaluate {current_currency} price in {target_currency}")
+        raise errors.MissingPriceDataError(
+            f"no price data to evaluate {current_currency} price in {target_currency}")
+
+    def _try_converting_with_multiple_pairs(
```

Comment:
💯

Reply:
Can we name it `try_convert_currency_value_using_multiple_pairs` to be consistent with `convert_currency_value_using_last_prices` and make it public please ? I think it will be very useful

Reply:
Can we add a `return None` at the end to explicitely return None when no value is available plz ?

Reply:
done